### ISSUE TITLE
Connection nodes together

### ISSUE THREAD

**Issue by victorb** [EverythingStays/EverythingStays#2]
I think there is a need of having a project (existing or creating a new one) for having a group of nodes that you can automatically tell to seed your module open publish. I imagine a workflow like this:

``` bash
npm publish
Publishing to IPFS...
Added to IPFS under /ipfs/QmaSDweaada
Pushing to nodes [0/12] completed
Pushing to nodes [4/12] completed
...
Done pushing to node! [1/12] failed
```

This would need work in two areas. The CLI should be able to push IPFS hashes to instances and show status for the user. The Node should be able to listen for requests, pin content based on that with a whitelist of ipfs ids (for private nodes, public nodes have a blacklist) and respond when done.

**Comment by NoraCodes** [comment]
This is a problem being solved by the IPFS project with FileCoin.

**Comment by victorb** [comment]
@SilverWingedSeraph Not really. I mean, Filecoin will help out getting people interested in seeding other peoples data, but the code still has to be written/integrated in the CLI and in the node. Filecoin is not gonna get code written for us :)

**Comment by NoraCodes** [comment]
It will solve this specific problem, though, won't it? Yes, we'll have to add a "publish to IPFS" command, but we won't have to have it push to additional nodes. Once IPFS is integrated with Filecoin, any data put on the network will be automatically picked up by FC miners as long as you pay a little for it.

**Comment by victorb** [comment]
@SilverWingedSeraph no, I don't think so. This specific problem is about connecting your publish with nodes that will seed. Filecoin will solve the problem of finding those nodes, but not connecting them.

I also don't expect Filecoin to automatically seed content, but I guess we don't know at this point.

Point being that today, Filecoin doesn't exist and I think we need to make it possible for people to host and offer nodes to seed modules in the network.
